examples/megasparse/README.md

```diff
@@ -0,0 +1,76 @@
+
+
+## <div>`INTRO`</div>
+
+<samp>
+
+<div>
+MegaSparse allows you to quickly serve a streamlit app running on top of the DeepSparse Server for comparing latency speeds of various transformer models. Currently, the demo can safely load up to 18 Sparse BERT models fine-tuned on the question answering task on only 16GB of RAM. The purpose of this app is for you to familiarize and compare the inference performance of various models trained on different sparse profiles.
+</div>
+
+<br />
+
+[Getting Started with the DeepSparse Server](https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/server)🔌
+
+<br />
+
+![alt text](./img/demo_screenshot.png)
+
+<br />
+
+## <div>`INSTALL`</div>
+
+```bash
+git clone https://github.com/neuralmagic/deepsparse.git
+```
+
+```bash
+cd examples/megasparse
+```
+
+```bash
+pip install -r requirements.txt
+```
+<br />
+The `config.yaml` file lists the models you want the DeepSparse Server to serve. Currently, it contains up to 18 BERT models, although 16 of these models are commented out so you can safely get started on two models, and later on, add more models at your own pace. If you wish to load all 18 models into memory make sure you have 16GB of RAM available, otherwise you will get out of memory errors. Also, as you add additional models to the `config.yaml` file, make sure to all also add a `MultiPipelineClient` object to the `variants` attribute in the `settings.py` module.
```

Comment:
Make sure to mention the config is actually at `server/config.yaml` so people can find it. 

Also we could make separate files no? One that is all 18 as an "intensive scenario" and one that is the few we want to highlight as default, which I'm thinking could be
```
zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/base-none
zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/12layer_pruned80_quant-none-vnni
zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned_quant_6layers-aggressive_96
zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned_quant_3layers-aggressive_89
```
@markurtz what do you think?

Reply:
oh nice, yea that's better